modules/configuring/snip-provisioning-your-custom-configuration-next-steps-in-default-context.adoc

```diff
@@ -0,0 +1,8 @@
+.Next steps
```

Comment:
technically speaking, these are presented additional steps. according to the modularization guide, `Next steps` should only provide a list of resources: https://redhat-documentation.github.io/modular-docs/#next-steps